code/modules/hydroponics/hydroponics.dm

```diff
@@ -521,6 +521,12 @@
 		mutmod = 0
 		adjustNutri(round(S.get_reagent_amount("robustharvestnutriment") *1 ))
 
+
+	//Fish Water is both an excellent fertilizer and waters
+	if(S.has_reagent("fishwater", 1))
```

Comment:
The old botany used a big `switch` and repeated `if` statements. It just wasn't as large because it didn't have as many reagents: https://github.com/ParadiseSS13/Paradise/blob/f1c3a7cd13510ecfd99137835ad6ffbe3ffe86de/code/modules/hydroponics/trays/tray_reagents.dm#L56